api/client/clientRouter.js

```diff
@@ -268,4 +275,138 @@ router.delete('/:id/notes/:id', async (req, res) => {
   }
 });
 
+/*
+Intake Form Router
+*/
+
+router.get('/:id/intake', (req, res) => {
+  Clients.findById(req.params.id)
+    .then((client) => {
+      const household = Household.findHouseholdbyId(client['household_id']);
+      const locations = Household.findLocationByHouseholdId(
+        client['household_id']
+      );
+      const gender = Clients.getGenderIdByClientId(req.params.id);
+      const race = Clients.getRaceIdByClientId(req.params.id);
+      const ethnicity = Clients.getEthnicityByClientId(req.params.id);
+      const phone_numbers = Clients.getPhoneNumbersByClientId(req.params.id);
+      const email_addresses = Clients.getEmailByClientId(req.params.id);
+      const education_histories = Clients.getEducationHistoryByClientId(
+        req.params.id
+      );
+      const output =
+        client +
+        household +
+        locations +
+        gender +
+        race +
+        ethnicity +
+        phone_numbers +
+        email_addresses +
+        education_histories;
+      res.status(200).json({
+        output,
+      });
+    })
+    .catch(() => {
+      res.status(500).json({
+        message: 'ERROR!',
+      });
+    });
+});
+
+router.put('/:clientId/intake/gender', async (req, res) => {
```

Comment:
What is the reason for making the route have intake? Could it just be `/:clientId/gender`?